### PR TITLE
Add e_machine values

### PR DESCRIPTION
**Pull request by eShuttleworth**
I've run into a few cases in which rabin2 didn't recognize the e_machine value in an elf header, this PR adds a bunch from http://www.sco.com/developers/gabi/latest/ch4.eheader.html, which was the most complete source I could find.

### CODE REVIEW

File: libr/bin/format/elf/elf.c [hunk 1]
```diff
@@ -2135,8 +2135,9 @@ char* Elf_(r_bin_elf_get_arch)(ELFOBJ *bin) {
 	case EM_LANAI:
 		return strdup ("lanai");
 	case EM_VIDEOCORE3:
-	case EM_VIDEOCORE4:
-		return strdup ("vc4");
```
Comment (radare) [inline libr/bin/format/elf/elf.c]: Theres no vc3 plugin. Why did you removed the vc4 one? We have no tests but this was working fine

File: libr/bin/format/elf/elf.c [hunk 2]
```diff
@@ -2149,90 +2150,166 @@ char* Elf_(r_bin_elf_get_arch)(ELFOBJ *bin) {
...
-	case EM_OPENRISC:    return strdup ("OpenRISC 32-bit embedded processor");
-	case EM_ARC_A5:      return strdup ("ARC Cores Tangent-A5");
-	case EM_XTENSA:      return strdup ("Tensilica Xtensa Architecture");
-	case EM_AARCH64:     return strdup ("ARM aarch64");
-	case EM_PROPELLER:   return strdup ("Parallax Propeller");
-	case EM_MICROBLAZE:  return strdup ("Xilinx MicroBlaze");
-	case EM_RISCV:       return strdup ("RISC V");
-	case EM_VIDEOCORE3:  return strdup ("VideoCore III");
-	case EM_VIDEOCORE4:  return strdup ("VideoCore IV");
+	case EM_NONE:          return strdup ("No machine");
```
Comment (radare) [inline libr/bin/format/elf/elf.c]: Maybe good to sort alphabetically

File: libr/bin/format/elf/elf.c (hunk 1 above)
Comment (eShuttleworth) [reply libr/bin/format/elf/elf.c]: Whoops, that's leftover from when I was first figuring out how this worked.  Fixed.

File: libr/bin/format/elf/elf.c (hunk 2 above)
Comment (eShuttleworth) [reply libr/bin/format/elf/elf.c]: The ordering is by the actual value of the E_MACHINE defines, eg EM_NONE=0, EM_M32=1, etc.  This how it was originally ordered, and I don't think it makes much sense to sort in any other way.

**Pull request merged**